### ISSUE TITLE
[BUG] - can't get to Babbage PV8 on node with preliminary Conway support

### ISSUE THREAD

**Issue by mkoura** [input-output-hk/cardano-node#4978]
*Internal*

**Area**
*Other*   Any other topic (Delegation, Ranking, ...).

**Summary**
On a node with preliminary Conway support, I can't get to era with Babbage protocol version 8.
The issue happens only on local cluster that updates to PV8 using update proposal. Local cluster that uses `TestBabbageHardForkAtEpoch` works ok.

It doesn't make any difference if I set `TestEnableDevelopmentHardForkEras` to `true` or not. There is a difference in behavior depending on `TestEnableDevelopmentNetworkProtocols` settings.

With `TestEnableDevelopmentNetworkProtocols: true`, I can reach epoch where protocol version is supposed to be set to 8, but have following issue:

```
cardano-cli query protocol-parameters --testnet-magic 42
Command failed: query protocol-parameters   Error: Consensus mode and era mismatch. Consensus mode: CardanoMode Era: ConwayEra
```

Without `TestEnableDevelopmentNetworkProtocols: true`, after reaching the epoch where protocol version is supposed to be set to 8, no more blocks are produced and the chain is not progressing. I have following errors in log files:

```
[bender-3:cardano.node.ErrorPolicy:Error:59] [2023-03-13 11:18:42.70 UTC] IP 127.0.0.1:41449 ErrorPolicyUnhandledApplicationException (HardForkEncoderDisabledEra (SingleEraInfo {singleEraName = "Conway"}))
```

**System info:**
- OS Name: Fedora 36
- Node version: bdd471932337ff689c1d85c543af36c20a9670e1
- CLI version: bdd471932337ff689c1d85c543af36c20a9670e1

**Comment by elricWu** [comment]
https://rhizome.cloud may help you. It's a ADA public node.

**Comment by mkoura** [comment]
It is already fixed in 8.0.0 and master.
